### ISSUE TITLE
Inaccessible "About OpenSpace" dialog

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3108]
When the "About OpenSpace" button is activated, the overlay is displayed covering the  space viewer. The required mark-up is not implemented to convey the container as a  modal dialog, and interaction with the overlay is problematic for users of access  technologies: 
- The focus indicator is not visible once the dialog appears, and users must press  Shift+Tab to reach the "Close" button.  
- Screen reader users encounter complete silence in the speech output and the result of their action remains unclear. To access the overlay, users must navigate  backwards and read through the items in reverse order. 
-  Both screen reader and keyboard users can move focus onto elements outside of  the displayed content, and into the application toolbars which are now obscured.  

Note that the dialog heading is correctly conveyed as a level 1 heading to emphasize  the current context of a modal interface.

**Suggested Solution** 
Implement an accessible modal dialog for the "About OpenSpace" overlay. Wrap its  container in a native <dialog> element which comes with several built-in accessibility  features: 
- provides programmatic boundaries for the overlay; 
- sends focus to the dialog when displayed on its first interactive element, the "Close"  button; 
- prevents users from reaching the dimmed main page content when in view; and 
- returns the focus back to the triggering control when dismissed, with the note that  in this case, focus should go back to the collapsed menu button.  

As an additional adjustment, apply an aria-labelledby attribute to the <dialog> element,  referencing the ID of the modal heading to provide an accessible name for the overlay. 

Severity: 1

**Comment by engbergandreas** [comment]
We should make this a link to the website and remove the screen alltogether

**Comment by engbergandreas** [comment]
Closed With https://github.com/OpenSpace/OpenSpace/commit/e3f5b83f7476c8507dccd45ab98e4731504af532
